ratis-server/src/main/java/org/apache/ratis/server/RaftServer.java

```diff
@@ -71,6 +72,8 @@ default RaftPeer getPeer() {
     StateMachine getStateMachine();
 
     DataStreamMap getDataStreamMap();
+
+    RaftClient getRaftClient();
```

Comment:
@szetszwo it seems that we no longer use this method to submit request. If it's the case, I'm willing to figure out whether we can drop this "dead code".

Reply:
You are right that we may remove it.  But we need to keep it in branch-2 for compatibility.

Reply:
OK. I submitted #886 already that is against master (3.0). I'll file a JIRA ticket later.